manim/mobject/mobject.py

```diff
@@ -1337,20 +1369,30 @@ def apply_function_to_submobject_positions(self, function: MappingFunction) -> S
             submob.apply_function_to_position(function)
         return self
 
-    def apply_matrix(self, matrix, **kwargs) -> Self:
+    def apply_matrix(
+        self,
+        matrix,
```

Comment:
Could you please type `matrix`, now that we're here?
```suggestion
        matrix: MatrixMN,
```